### ISSUE TITLE
Support for NVMe Namespaces

### ISSUE THREAD

**Issue by mateoconlechuga** [Drive-Trust-Alliance/sedutil#371]
Is there currently support for NVMe namespaces, as described in this document? https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage_Configurable_Namespace_Locking-_NS_and_LUN_v1p01_r1p11_31mar2021.pdf 

I'm curious if I can configure locking regions for individual namespaces and how that works within the scope of the LBA regions defined by the Opal specification.

**Comment by r0m30** [comment]
I have not looked into namespace locking at all. Namespaces have interesting possibilities but at this point it's low on the priority list.

**Comment by mateoconlechuga** [comment]
That's all right; just curious! It amazes me how many specifications on top of other specifications there are -- it's rather insane.

**Comment by tonsui** [comment]
Under Windows, you need to use SCSI sendCmd, lukefor has implementation in this fork: https://github.com/lukefor/sedutil/tree/windows_nvme
